client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -96,6 +96,7 @@ defineProps<Props>();
                         <div id="history_id">
                             {{ dataset.history_id }}
                             <DecodedId :id="dataset.history_id" />
+                            (<a :href="withPrefix(`/histories/view?id=${dataset.history_id}`)">view</a>)
```

Comment:
```suggestion
                            (<RouterLink :to="`/histories/view?id=${dataset.history_id}`">view</RouterLink>)
```
I would suggest using a router link (would also need the import at the top ofc) because it makes for better UX as there is an instant route change instead of a page load?

Reply:
Thanks! I know absolutely nothing about client development so was just inferring based on other patterns I found.

Reply:
Thank you @natefoo!!! 😄 
Is it ok if I push one tiny commit with the import at the top?:
```typescript
import { RouterLink } from "vue-router";
```

Reply:
Any time, :shipit: